### PR TITLE
B lorient657

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient657.xml [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+</decoNote>
+</decoDesc>
+
+    <additions>
+    <list>
+    <item xml:id="a1">
+    <locus from="2v"/>
+    <desc type="GuestText"><title type="complete">A short chronological section from Adam to Christ, in Amharic.</title></desc>
+    <q xml:lang="am"><seg part="F">ዘንተ፡ ዘአጽሐፈ፡ <persName role="patron" ref="PRS13283SahlaMikael">ሣህለ፡ ሚካኤል፡</persName> <placeName ref="LOC5662Semada">ዘስማዳ</placeName>፡ ወዘጸሐፈ፡
+    <persName role="scribe" ref="PRS13284GabraIyasus">ገብረ፡ ኢየሲስ፡</persName> ዘ<placeName ref="LOC6563Sada">ፀዳ፡</placeName> ዘተቀብረ፡ ቦቱ፡ ጻድቅ፡ ዮሐንስ፡ ንጉሠ፡ ነገሥት፡
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient657.xml:126]: can the name of the emperor also be marked up?

File: LondonBritishLibrary/orient/BLorient657.xml [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+  <locus target="#2r #220v #221r"/>
+  <desc>Hymns to John the Baptist.</desc>
+  </item>
+
+<item xml:id="a4">
+<locus target="#218"/>
+<desc>F. 218 is a later insertion containing the commemoration of king <persName ref="PRS10375Yohanne">John I</persName>, 15th of Ḥamle. It bears the name of <persName ref="PRS13286WaldaMaryam">Walda Māryām</persName> and
+his father <persName ref="PRS13285HabtaMaryam">Habta Māryām</persName>.</desc>
+</item>
+
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient657.xml:147]: This means that the manuscript is a composite one. You wpuld need to rework the record and add ms_Parts, and occasionally sort out information pertaining to the whole mansucript or to its parts

**Review by DariaElagina — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient657.xml [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+                      </item>
+
+                    <item xml:id="e1">
+                    <locus target="#2r"/>
+                    <desc>The name of ʾabbā Gabra ʾIyasus is on <locus target="#2r">f. 2a</locus>.</desc>
+                    </item>
+
+                      </list>
+                    </additions>
+                      </physDesc>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient657.xml:101]: I suppose that almost all the information which is above belongs to the description of the first part.

File: LondonBritishLibrary/orient/BLorient657.xml [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+
+                      <history>
+                        <origin>
+                            <origDate notAfter="1667"/>
+                            <desc>It is the second part of the Synaxarium described in <ref type="mss" corresp="BLorient657"/>.</desc>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient657.xml:106]: This information also belongs to the first part, I suppose. Concerning the date it depends on where the dating is provided.

**Review by DariaElagina — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient657.xml [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+    <locus target="#4r"/>
+    <desc type="OwnershipNote">The note states that the manuscript belongs to the church of <placeName ref="INS0101MadhaneAlam">Madḫāne ʿAlam</placeName>
+    </desc>
+    <q xml:lang="gez">ስንክሳር፡ ዘቅዱስ፡ መድኃኔ፡ ዓለም።</q>
+  </item>
+
+  <item xml:id="a3">
+  <locus target="#2r #220v #221r"/>
+  <desc>Hymns to John the Baptist.</desc>
+  </item>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient657.xml:169]: please, add the type

File: LondonBritishLibrary/orient/BLorient657.xml [hunk 6]
```diff
@@ -0,0 +1,256 @@
...
+
+  <item xml:id="e1">
+  <locus target="#2r"/>
+  <desc>The name of ʾabbā Gabra ʾIyasus is on <locus target="#2r">f. 2a</locus>.</desc>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient657.xml:173]: Could you, please, create a person record for him. You do not need to write out locus (f. 2a), if it is encoded, the string will be print out automatically.

File: LondonBritishLibrary/orient/BLorient657.xml (hunk 5 above)
Comment (MarcinKrawczuk) [reply LondonBritishLibrary/orient/BLorient657.xml:169]: Since the mss. is online, let's to this properly

**Review by DariaElagina — APPROVED** [review]

**Pull request merged**